Tasks/VSMobileCenterUpload/task.json

```diff
@@ -0,0 +1,149 @@
+            "visibleRule": "symbolsType = Windows || symbolsType == AndroidNative"
+        },
+        {
+            "name": "dsymPath",
+            "type": "filePath",
+            "label": "dSYM Path",
+            "groupName": "symbols",
+            "required": false,
+            "helpMarkDown": "Relative path from the repo root to dSYM folder.",
+            "visibleRule": "symbolsType = Apple"
+        },
+        {
+            "name": "mappingTxtPath",
+            "type": "filePath",
+            "label": "Mapping File",
+            "groupName": "symbols",
+            "required": false,
+            "helpMarkDown": "Relative path from the repo root to Android's mapping.txt file.",
+            "visibleRule": "symbolsType = AndroidJava"
+        },
+        {
+            "name": "packParentFolder",
+            "type": "boolean",
+            "label": "Include All Items in Parent Folder",
+            "groupName": "symbols",
+            "required": false,
+            "helpMarkDown": "Upload the selected symbols file / folder and every other items inside the same parent folder."
+        },
+        {
+            "name": "releaseNotesSelection",
+            "type": "radio",
+            "label": "Create Release Notes",
+            "required": true,
+            "defaultValue": "input",
+            "options": {
+                "input": "Enter Release Notes",
+                "file": "Select Release Notes File"
+            }
+        },
+        {
+            "name": "releaseNotesInput",
+            "type": "multiLine",
+            "label": "Release Notes",
+            "required": true,
+            "helpMarkDown": "Release Notes for this version.",
+            "visibleRule": "releaseNotesSelection = input"
+        },
+        {
+            "name": "releaseNotesFile",
+            "type": "filePath",
+            "label": "Release Notes File",
+            "required": true,
+            "helpMarkDown": "Select a UTF-8 encoded text file which contains the Release Notes for this version.",
+            "visibleRule": "releaseNotesSelection = file"
+        },
+        {
+            "name": "distributionGroupId",
+            "type": "string",
+            "defaultValue": "",
+            "label": "Distribution Group Id",
+            "helpMarkDown": "Id of the distribution group app will deploy to, leave it empty for the default group.",
+            "required": false
+        }
+    ],
+    "instanceNameFormat": "Deploy $(app) to Visual Studio Mobile Center",
+    "execution": {
+        "Node": {
+            "target": "vsmobilecenterupload.js",
+            "argumentFormat": ""
+        }
+    },
+    "messages": {
+        "CannotDecodeEndpoint": "Could not decode the endpoint.",
+        "NoResponseFromServer": "No response from serve.",
+        "FailedToUploadFile": "Failed to complete file upload.",
+        "NoApiTokenFound": "No API token found on Sonoma Connection.",
+        "Succeeded":"Task succeeded"
```

Comment:
Might be useful to include the task name or something like "VS Mobile Center Upload Task succeeded"